kbase-extension/static/kbase/js/widgets/function_output/kbaseModelCoreNarrative.js

```diff
@@ -138,56 +132,54 @@
                 }
             }
 
-            function core_model(data, show_flux) {
+            function core_model(data) {
                 const stage = Raphael('core-model', 1000, 1500);
 
                 // Draw shapes
-                for (var i in data) {
-                    var obj = data[i];
-
+                for (const i in data) {
+                    const obj = data[i];
+                    let text;
                     const x = (obj.x - 2) * grid; //Fixme: update json
                     const y = (obj.y - 2) * grid;
 
                     if (obj.shape == 'rect') {
-                        const rect = stage.core_rect(obj, x, y, r_width, r_height, show_flux);
-                        //var text = stage.text(x+r_width/2+50, y+r_height/2, obj.rxns.join('\n'))
-                        //text.attr({'font-size':8})
+                        // rect = stage.core_rect(obj, x, y, r_width, r_height, show_flux);
```

Comment:
These vars never get used so commented out (for now). Who knows what is going on in this code... maybe they will be useful to a hypothetical future code reader.